lib/src/features/calls/call_link/call_link.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+
+class CallLinkPage extends StatelessWidget {
+  const CallLinkPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Create Call Link"),
+      ),
+      body: SingleChildScrollView(
+        padding: EdgeInsets.all(16),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(bottom: 16),
+              child: Text(
+                "Anyone with WhatsApp can use this link to join this call. Only share it with people you trust.",
```

Comment:
You can create separate class file for Strings and create variables for them. You can access them here